tests/elastio_snap.py

```diff
@@ -103,14 +104,18 @@ def reload_incremental(minor, device, cow_file, cache_size=0, ignore_snap_errors
     return 0
 
 
-def destroy(minor):
-    ret = lib.elastio_snap_destroy(minor)
-    if ret != 0:
-        return ffi.errno
-
-    util.settle()
-    return 0
+def destroy(minor, retries=3, debug=True):
+    for retry in range(retries):
```

Comment:
The entire Elastio architecture is built on retries.

Reply:
I would go further and say any modern software of any complexity worked only because of carefully placed retried.